exercises/concept/freelancer-rates/test/freelancer_rates_test.exs

```diff
@@ -83,7 +83,7 @@ defmodule FreelancerRatesTest do
     @tag task_id: 4
     test "it applies the discount" do
       # 1.25
-      assert FreelancerRates.days_in_budget(480, 60, 20) == 1.2
+      assert FreelancerRates.days_in_budget(480, 60, 20) == 0.8
```

Comment:
I can't find an issue in this test.

budget = 480, hourly_rate = 60, discount = 20%
daily_rate = 8 * 60 = 480
daily_rate_with_discount = 480 - 20% = 384
days_in_budget = 480 / 384 = 1.25 (rounded to 1.2)

Can you point to the problem?

Reply:
You are correct, my bad